model/map-hot-rod/src/main/java/org/keycloak/models/map/storage/hotRod/userSession/HotRodUserSessionTransaction.java

```diff
@@ -65,6 +66,7 @@ public void commit() {
     }
 
     private MapAuthenticatedClientSessionEntity wrapClientSessionEntityToClientSessionAwareDelegate(MapAuthenticatedClientSessionEntity d) {
+        if (!clientSessionTransaction.exists(d.getId())) return null;
```

Comment:
These changes should be in the followup PR.

Reply:
done